packages/sdk-core/src/model/report/BacktraceReport.ts

```diff
@@ -109,4 +110,8 @@ export class BacktraceReport {
             this.classifiers.unshift(...options.classifiers);
         }
     }
+
+    private generateErrorMessage(data: unknown) {
+        return typeof data === 'object' ? JSON.stringify(data, jsonEscaper()) : data?.toString() ?? '';
```

Comment:
Should we maybe make the default something like 'Unknown error message'?

Reply:
Can we leave it for now as it is and then adjust the flow when we recognize customers want to have something else? If someone puts undefined or null for any purpose - it would be better to show an empty message to show something went wrong

Reply:
Works for me!